exercises/J-array-get-set/exercise.js

```diff
@@ -5,11 +5,11 @@
 */
 
 function first(arr) {
-  return; // complete this statement
+  return arr [0]; // complete this statement
```

Comment:
Usually, the `[]` to denote indexing are placed right next to the variable, without the space (e.g. `arr [0] -> arr[0]`). I didn't actually know that it works with the space to be honest :sweat_smile: But it's a convention you'll see everywhere (and I believe many other languages enforce it)